specification/resources/resource-manager/Microsoft.Management/2017-11-01-preview/management.json

```diff
@@ -0,0 +1,708 @@
+                    "readOnly": true
+                },
+                "type": {
+                    "type": "string",
+                    "description": "The type of the resource.  For example, /providers/Microsoft.Management/managementGroups",
+                    "readOnly": true
+                },
+                "name": {
+                    "type": "string",
+                    "description": "The name of the management group. For example, 00000000-0000-0000-0000-000000000000",
+                    "readOnly": true
+                },
+                "properties": {
+                    "title": "Properties",
+                    "$ref": "#/definitions/ManagementGroupProperties"
+                }
+            }
+        },
+        "ManagementGroupProperties": {
+            "description": "The generic properties of a management group.",
+            "type": "object",
+            "properties": {
+                "tenantId": {
+                    "type": "string",
+                    "format": "uuid",
+                    "description": "The AAD Tenant ID associated with the management group. For example, 00000000-0000-0000-0000-000000000000"
+                },
+                "displayName": {
+                    "type": "string",
+                    "description": "The friendly name of the management group."
+                },
+                "details": {
+                    "title": "Details",
+                    "$ref": "#/definitions/ManagementGroupDetails"
+                },
+                "children": {
+                    "description": "The list of children.",
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/ManagementGroupChildInfo"
+                    }
+                }
+            }
+        },
+        "ManagementGroupDetails": {
+            "description": "The details of a management group.",
+            "type": "object",
+            "properties": {
+                "version": {
+                    "type": "number",
+                    "format": "int32",
+                    "description": "The version number of the object."
+                },
+                "updatedTime": {
+                    "type": "string",
+                    "format": "date-time",
+                    "description": "The date and time when this object was last updated."
+                },
+                "updatedBy": {
+                    "type": "string",
+                    "description": "The identity of the principal or process that updated the object."
+                },
+                "parent": {
+                    "title": "Parent",
+                    "$ref": "#/definitions/ParentGroupInfo"
+                }
+            }
+        },
+        "ManagementGroupChildInfo": {
```

Comment:
Please add id, name and type top level properties for this.

Reply:
Removing GET for now to match private preview API release. GET for the subscription under management group will be added in the next version of the API.

Reply:
@ravbhatnagar Mind confirming whether or not this is sufficient?